documentation/decisions/adr/00NN-deployment-service.md

```diff
@@ -0,0 +1,99 @@
+# Deployment Strategy
+
+- **Status:** Accepted <!-- REQUIRED -->
+- **Last Modified:** 2023-07-20 <!-- REQUIRED -->
+- **Related Issue:** [187](https://github.com/HHS/grants-equity/issues/187) <!-- RECOMMENDED -->
+- **Deciders:** Lucas Brown, Sammy Steiner, Billy Daly <!-- REQUIRED -->
+- **Tags:** ADR <!-- OPTIONAL -->
+
+## Context and Problem Statement
+
+We need to choose a deployment strategy for the Grants.gov modernization effort that suits our core needs and will host our entire application ecosystem, both the API and frontend layers.
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+- **Reliable:** The chosen deployment strategy should be able to offer uninterrupted application uptime with consistent results.
+- **Scalable:** We should be able to scale our deployment to meet the demands of our containers while remaining lean.
+- **Compatible with IaC:** The strategy should be compatible with Terraform as our chosen Infrastructure as Code solution.
+- **Ease of use:** We prioritize ease of use as well as cost-efficiency. We understand that engineering time spent managing details of infrastructure is a trade off to less prescriptive strategies.
+- **Growth oriented:** As our applications evolve, we need our chosen deployment strategy to grow with us or be easily swapped.
+
+## Options Considered
+
+- ECS with Fargate or EC2 launch type
+- S3
+- Lambda
+
+## Decision Outcome <!-- REQUIRED -->
+
+Chosen option: **ECS with Fargate launch type**, because it offers the most consistent and easy to use deployment strategy to host both the front-end and API layers of the Grants.gov modernization. Current template infrastructure integrates with ECS and the Fargate launch type.
+
+## Pros and Cons of the Options <!-- OPTIONAL -->
+
+### ECS with Fargate or EC2 launch type
+
+#### Fargate
+
+Fargate is an AWS serverless compute tool for containers.
+
+- **Pros**
```

Comment:
Our use case (website and api) is perfect for `fargate spot` instance types which would further reduce our costs significantly. Maybe add that to the pros section?